scripts/tooling.py

```diff
@@ -14,10 +14,17 @@ def _run(cmd: str):
         sys.exit(r)
 
 
+def format():
+    _run(f"isort --profile=black .")
+    _run("black .")
+
+
 def lint():
```

Comment:
Just to ensure I understand it correctly, you envision `format` to be called locally and `lint`/`check` to be called from CI, right? (hence the `--check` flags)